packages/net/_test.pony

```diff
@@ -669,3 +673,124 @@ class _TestTCPProxyNotify is TCPConnectionNotify
 
   fun ref connect_failed(conn: TCPConnection ref) =>
     _h.fail_action("sender connect failed")
+
+class _TestTCPConnectionFailed is UnitTest
+  fun name(): String => "net/TCPConnectionFailed"
+
+  fun ref apply(h: TestHelper) =>
+    h.expect_action("connection failed")
+
+    let host = "127.0.0.1"
+    let port = "12345"
```

Comment:
to amuse me can we use `7669` for the port? That on a phone becomes "PONY"

Reply:
Done